File: create_dirs.py
#!/usr/bin/env python
import os

base_path = r'C:\github-ai-assisted-coding-2153871\summarizer-app'

# Create directories
dirs_to_create = [
    os.path.join(base_path, 'backend', 'app', 'summarizer'),
    os.path.join(base_path, 'backend', 'tests'),
    os.path.join(base_path, 'frontend', 'templates')
]

for dir_path in dirs_to_create:
    os.makedirs(dir_path, exist_ok=True)
    print(f'Created: {dir_path}')

# Verify they exist
print('\nVerifying directories exist:')
for dir_path in dirs_to_create:
    exists = os.path.isdir(dir_path)
    status = '✓' if exists else '✗'
    print(f'{status} {dir_path}')

File: create_dirs.js
const fs = require('fs');
const path = require('path');

const basePath = 'C:\\github-ai-assisted-coding-2153871\\summarizer-app';

const dirsToCreate = [
    path.join(basePath, 'backend', 'app', 'summarizer'),
    path.join(basePath, 'backend', 'tests'),
    path.join(basePath, 'frontend', 'templates')
];

dirsToCreate.forEach(dirPath => {
    fs.mkdirSync(dirPath, { recursive: true });
    console.log(`Created: ${dirPath}`);
});

console.log('\nVerifying directories exist:');
dirsToCreate.forEach(dirPath => {
    const exists = fs.existsSync(dirPath);
    const status = exists ? '✓' : '✗';
    console.log(`${status} ${dirPath}`);
});
